src/number_theory/fermat4.lean

```diff
@@ -168,7 +168,7 @@ begin
   have ht : pythagorean_triple (a ^ 2) (b ^ 2) c,
   { delta pythagorean_triple,
     have H := h.1.2.2,
-    linear_combination H },
+    linear_combination (H) },
```

Comment:
```suggestion
    linear_combination (h.1.2.2) },
```
This should work now, right?

Is it even smart enough to see through the `pythagorean_triple`, so that one can delete the `delta pythagorean_triple` line?

Reply:
Yes, this should work. It won't see through the `pythagorean_triple` line in the goal -- it uses the goal to infer the type of the equality (needed for the empty `linear_combination` to work).